runtimes/peregrine/src/lib.rs

```diff
@@ -1201,11 +1200,6 @@ pub type Executive = frame_executive::Executive<
 	Runtime,
 	// Executes pallet hooks in the order of definition in construct_runtime
 	AllPalletsWithSystem,
```

Comment:
I think we have to add a migration to remove the pallet, right?

Reply:
Good catch: https://github.com/KILTprotocol/kilt-node/pull/723/commits/8057f3be0689bfe14eb9ea8eb781b64ef25f098f.